src/Symfony/Component/ObjectMapper/ArrayMapper.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\ObjectMapper;
+
+use Symfony\Component\ObjectMapper\Exception\MappingException;
+use Symfony\Component\ObjectMapper\Exception\WrappedMappingException;
+
+/**
+ * Map an array of objects into the specified target array using the ObjectMapper.
+ *
+ * @experimental
```

Comment:
Reading the code, I understand that this class is not meant to be used directly. If that's the case, it should be marked as internal.

```suggestion
 * @internal
```